README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
+
```

Comment:
Consider adding a brief description of what 'task install' and 'task dev' do, especially for users unfamiliar with these commands. This can improve the readability and maintainability of the script by providing context.